src/pages/Main/ProductList/ProductList.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import Product from '../Product/Product';
+import './ProductList.scss';
+
+const ProductList = props => {
+  const { cateId, filteredProducts, limit, location, setIsOpenModal } = props;
+  const loop = () => {
+    const result = [];
+    for (let i = 0; i < 3; i++) {
+      result.push(
+        <button>
+          <Link to={`/cateCode/${cateId}?offset=${i * 10}&limit=${limit}`}>
+            {i + 1}
+          </Link>
+        </button>
+      );
+    }
+    return result;
+  };
+
+  let title = '';
+  switch (cateId) {
+    case 0:
+      title = '전체상품';
+      break;
+    case 1:
+      title = '여성 영양제';
+      break;
+    case 2:
+      title = '남성 영양제';
+      break;
+    case 3:
+      title = '아이 영양제';
+      break;
+    default:
+      title = '전체상품';
+      break;
+  }
```

Comment:
```suggestion
  const listTitle = {
    1: '여성 영양제',
    2: '남성 영양제',
    3: '아이 영양제',
  };
```

switch 문 대신 객체를 활용해서 title 값을 관리할 수 있겠습니다!